docs/getting-started/run-applications.md

```diff
@@ -33,12 +33,38 @@ on-demand by Crossplane.
 > published a PostgreSQLInstance with at least one working Composition in order
 > to create the OAM application we'll use in this guide.
 
-## Install the OAM Addon
+## Infrastructure Operator
+
+### Install OAM build-in workloads and traits
 
 As the infrastructure operator our work is almost done - we defined, published,
 and composed the infrastructure  that our application developer and operator
-team-mates will use in the previous guide. One task remains for us, which is to
-install and configure Crossplane's OAM addon.
+team-mates will use in the previous guide. One task remains is, we must define 
+the [_workloads_] and [_traits_] that our platform supports. We'll discuss workloads 
+and traits in more detail later.
+
+The infrastructure operator could choose which workloads
+and traits their platform supports by creating or deleting `WorkloadDefinitions`
+and `TraitDefinitions` like below:
+
+```yaml
+---
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: containerizedworkloads.core.oam.dev
+spec:
+  definitionRef:
+    name: containerizedworkloads.core.oam.dev
+``` 
+Run the following command to add support for the build-in workloads and traits required
+by this guide:
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/master/docs/snippets/run/definitions.yaml
+```
+
```

Comment:
Can you help me understand why you feel that it's better to define workloads before installing the addon, rather than after?

Reply:
If you think `Definitions` as CRD (which is the intention), "addon" as controllers (which indeed are), it would be easier to get my point for this change.